posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -12,38 +12,42 @@ open-graph-image: https://openliberty.io/img/twitter_card.jpg
 
-The goals of LRA are: 
+In traditional applications, ACID is the well-known transaction model and the most commonly used atomic commit protocol is the two-phase commit. In an ACID transaction, tightly-coupled XA resources can be directed to "rollback" by a coordinating transaction manager. This model introduces resource locking and global rollback, which does not scale well and therefore is not suitable for microservice architecture. As a consequence, the link:https://developer.ibm.com/depmodels/microservices/articles/use-saga-to-solve-distributed-transaction-management-problems-in-a-microservices-architecture#saga[SAGA pattern] was defined for microservice transaction architecture. A SAGA is a sequence of transactions. If one transaction fails, the saga undoes all of the preceding transactions. It aims for eventual consistence. MicroProfile LRA is an implementation of the SAGA pattern.
 
-* to have no strong coupling between services
-* to simplify application error-handling when multiple services are running as part of single logical transaction
-* ensures the execution of application-provided compensating actions if an activity is cancelled
-* allow actions to finish early
+The goals of MicroProfile LRA are: 
 
-The LRA model relies on having compensating actions for all business interactions and ensures that when the activity ends either all the work is accepted or all will be compensated. How each activity gets compensated is down to the individual services, but the model defines what triggers a compensating action and when they are executed.
+* have no strong coupling between services
+* simplify application error-handling when multiple services are running as part of single logical transaction
+* ensure the execution of application-provided compensating actions if an activity is cancelled
+* allow actions to finish early.
+
+The LRA model relies on having compensating actions for all business interactions and ensures that when the activity ends, all the work is either accepted or will be compensated. How each activity gets compensated depends on the individual services, but the model defines what triggers a compensating action and when they are executed.
 
 == LRA in Open Liberty
 
 In Open Liberty there are 2 parts to the using LRA:
 
-* LRA Participants which are applications using MicroProfile LRA annotations that cause them to become involved in the LRA
-* LRA Coordinator which manages the LRA processing. It handles the initialisation of the LRA, enlisting in a LRA and the completion or compensating of a LRA. This is enabled through standard Liberty configuration.
+* LRA participants, which are applications that use MicroProfile LRA annotations that involve them in the LRA
+* the LRA coordinator, which manages the LRA processing. It handles the initialization of the LRA, the enlistment of services in the LRA, and the completion or compensating of the LRA. The LRA Coordinator is enabled through standard Liberty configuration.
```

Comment:
"LRA Coordinator" should be "LRA coordinator" to be consistent with the changes made

Reply:
fixed throughout